npm/angular/src/mount.ts

```diff
@@ -219,7 +219,9 @@ function initTestBed<T> (
   return componentFixture
 }
 
-@Component({ selector: 'cy-wrapper-component', template: '' })
+// if using the Wrapper Component (template strings), the component itself cannot be
+// a standalone component
+@Component({ selector: 'cy-wrapper-component', template: '', standalone: false })
```

Comment:
since template strings require a references to the actual component in the test module, the component cannot be a standalone component